docs/resources/mws_permission_assignment.md

```diff
@@ -63,3 +63,30 @@ resource "databricks_mws_permission_assignment" "add_admin_spn" {
   permissions  = ["ADMIN"]
 }
 ```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `workspace_id` - Databricks workspace ID
+* `principal_id` - Databricks ID of the user, service principal, or group. The principal ID can be retrieved using the SCIM API.
+* `permissions` - The list of workspace permissions to assign to the principal:
+  * `“USER”` - Can access the workspace with basic privileges.
+  * `“ADMIN”` - Can access the workspace and has workspace admin privileges to manage users and groups, workspace configurations, and more.
+
+## Import
+
+The resource `databricks_mws_permission_assignment` can be imported using the workspace id and principal id
+
+```bash
+terraform import databricks_mws_permission_assignment.this workspace_id|principal_id
```

Comment:
put quotes around `workspace_id|principal_id` - otherwise it will be treated as shell pipe